src/common/Makefile.am

```diff
@@ -103,10 +99,8 @@ libcommon_internal_la_SOURCES += \
 endif
 
 # used by RBD and FileStore
-if LINUX
```

Comment:
@trociny & @tchaikov this will not compile on Solaris however, I had originally added a if WITH_RBD around that file.